### PR TITLE
chore: add bot to update bind status

### PR DESCRIPTION
**Pull request by forcodedancing**

### CODE REVIEW

**Review by huihzhao — APPROVED** [review]

File: .github/workflows/update.yml [hunk 1]
```diff
@@ -0,0 +1,57 @@
+name: Update Token Bind Status
```
Comment (James-Chase788) [inline .github/workflows/update.yml:1]: #- [ ] - James chase  account holder```
```

```
```

File: .github/workflows/update.yml [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+          path: |
+            ~/go/pkg/mod
+            ~/.cache/go-build
+            ~/Library/Caches/go-build
+            %LocalAppData%\go-build
+          key: ${{ runner.os }}-go-${{ hashFiles('**/go.sum') }}
+          restore-keys: |
+            ${{ runner.os }}-go-
+
+      - name: Install Go
```
Comment (James-Chase788) [inline .github/workflows/update.yml:30]: James chase

Comment (James-Chase788) [reply .github/workflows/update.yml:30]: > James chase

File: .github/workflows/update.yml [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+      - run: go run main.go
+
+      - name: Commit files
+        id: commit-files
+        run: |
+          if [ -n "$(git status --porcelain README.md)" ]; then
+            git config --local user.email "github-actions[bot]@users.noreply.github.com"
+            git config --local user.name "github-actions[bot]"
+            git add README.md
```
Comment (James-Chase788) [inline .github/workflows/update.yml:46]: >

**Pull request merged**